bpf/bpf_lxc.c

```diff
@@ -939,7 +941,7 @@ __section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_NAT64) int tail_ipv6_to_ipv4(struct
 
 __section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_NAT46) int tail_ipv4_to_ipv6(struct __sk_buff *skb)
 {
-	union v6addr dp = LXC_IP;
+	union v6addr dp = {};
```

Comment:
I don't think we have to zero out these cases here, above and below. I mean we make sure to init them at the time when they are needed, no?